source/tutorials/importing.rst

```diff
@@ -200,71 +200,74 @@ Importing data
-  # Lines starting with '#' are ignored and can be used to create
-  # "comments" or even "comment out" entries
-  sample-1,$PWD/some/filepath/sample1_R1.fastq.gz,forward
-  sample-2,$PWD/some/filepath/sample2_R1.fastq.gz,forward
-  sample-1,$PWD/some/filepath/sample1_R2.fastq.gz,reverse
-  sample-2,$PWD/some/filepath/sample2_R2.fastq.gz,reverse
+  sample-id	forward-absolute-filepath	reverse-absolute-filepath
+  sample-1	$PWD/some/filepath/sample0_R1.fastq.gz	$PWD/some/filepath/sample1_R2.fastq.gz
+  sample-2	$PWD/some/filepath/sample2_R1.fastq.gz	$PWD/some/filepath/sample2_R2.fastq.gz
+  sample-3	$PWD/some/filepath/sample3_R1.fastq.gz	$PWD/some/filepath/sample3_R2.fastq.gz
+  sample-4	$PWD/some/filepath/sample4_R1.fastq.gz	$PWD/some/filepath/sample4_R2.fastq.gz
 
-Just like with ``fastq.gz``, the absolute filepaths in the manifest for any ``fastq`` files must be accurate. The following example illustrates a simple fastq manifest file for ``fastq`` single-end read data for one sample.
+Just like with ``fastq.gz``, the absolute filepaths in the manifest for any ``fastq`` files must be accurate. The following example illustrates a simple fastq manifest file for ``fastq`` single-end read data for two samples.
 
 ::
 
-  sample-id,absolute-filepath,direction
-  sample-1,$PWD/some/filepath/sample1_R1.fastq,forward
+  sample-id	absolute-filepath
+  sample-1	$PWD/some/filepath/sample1_R1.fastq
+  sample-2	$PWD/some/filepath/sample2_R1.fastq
 
-There are four variants of FASTQ data which you must specify to QIIME 2 when importing, and which are defined in the following sections.
+There are four variants of FASTQ data which you must specify to QIIME 2 when importing, and which are defined in the following sections. Since importing data in these four formats is very similar, we'll only provide examples for two of the variants: ``SingleEndFastqManifestPhred33V2`` and ``PairedEndFastqManifestPhred64V2``.
 
-SingleEndFastqManifestPhred33
-*****************************
+SingleEndFastqManifestPhred33V2
+*******************************
 
 In this variant of the fastq manifest format, the read directions must all either be forward or reverse. This format assumes that the `PHRED offset`_ used for the positional quality scores in all of the ``fastq.gz`` / ``fastq`` files is 33.
 
-SingleEndFastqManifestPhred64
-*****************************
+.. download::
+   :url: https://data.qiime2.org/2019.4/tutorials/importing/se-33.zip
+   :saveas: se-33.zip
 
-In this variant of the fastq manifest format, the read directions must all either be forward or reverse. This format assumes that the `PHRED offset`_ used for the positional quality scores in all of the ``fastq.gz`` / ``fastq`` files is 64. During import, QIIME 2 will convert the PHRED 64 encoded quality scores to PHRED 33 encoded quality scores. This conversion will be slow, but will only happen one time.
+.. download::
+   :url: https://data.qiime2.org/2019.4/tutorials/importing/se-33-manifest
```

Comment:
Sample names are formatted differently from one another in the manifest. Intentional?
```
sample.1	$PWD/se-33/sample1.fastq.gz
sample2	$PWD/se-33/sample2_S1_L001_R1_001.fastq.gz
```

Reply:
> Intentional?

Yep!

```
# 2019.1 version


# single-end PHRED 33 fastq manifest file for forward reads
sample-id,absolute-filepath,direction
sample.1,$PWD/se-33/sample1.fastq.gz,forward
sample2,$PWD/se-33/sample2_S1_L001_R1_001.fastq.gz,forward
```

I think the goal here was so that we had something to point out that said "your IDs can be whatever you might dream them to be..."

![image](https://user-images.githubusercontent.com/274668/56625930-61dfe700-65f4-11e9-8f0e-3b3661521a93.png)